File: Web/ClientApp/src/app/Models/UserModel.ts
export class UserModel {
    rememberMe: boolean;
    email: string;
    verificationCode: string;
    grantType = 'password';
    username: string;
    password = '';
    userId: string;
    role: string;
    fullName: string;
    bankSortCode: string;
    walletCodec: string;
    contactAddress: string;
    bankAccount: string;
    subAccountName: string;

    }


export class UserLogin {
        id: string;
        email: string;
        password: string;
        firstName: string;
        lastName: string;
        token?: string;
        roles: string;
      }
export class IdToken {
        sub: string;
        name: string;
        fullname: string;
        email: string;
        role: string | string[];
        configuration: string;
      }

File: Domain/Models/UserModel.cs
namespace Domain.Models
{
    public class UserModel
    {
        public string userId {get;set;}
        public string UserName{get;set;}
        public string LastName{get;set;}
        public string FirstName{get;set;}
        public string Email{get;set;}
        public string roles{get;set;}
        public string Password{get;set;}
        public string PhoneNumber{get;set;}
        public string ConfirmPassword{get;set;}
        public string RegistrationType{get;set;}
        public string bankAccount { get; set; }
        public string subAccountName { get; set; }
        public string bankSortCode { get; set; }
        public string walletCodec { get; set; }
        public string ContactAddress {get;set;}
        public double percentage_charge { get; set; }
        public string settlement_bank { get; set; }
    }
}